app/pub_sub/subscribers/site_activity.rb

```diff
@@ -0,0 +1,33 @@
+module Subscribers
+  class SiteActivity < ::Subscribers::Base
+    def site_created(event)
+      return if event.payload[:subject].nil? || event.payload[:subject].new_record?
```

Comment:
Regarding these checks before actually tracking anything, I'm wondering if we should just trust every message at this point so we're not adding any extra logic here. I mean to broadcast events only when things have been done instead of using callbacks that are executed regardless the result. How about placing the activity tracking methods (`track_<action>_activity`) inside every controller action, when the record is saved, deleted and so on?

Since we'll be adding some more complexity to these actions, I'd also consider abstracting these group or operations (i.e. get parameters + save the record + trigger callbacks) into service objects. Let's catch-up tomorrow if it's OK with you.